sites/search/json/canada-en.json

```diff
@@ -0,0 +1,487 @@
+{
+  "@context": {
+    "dc": "http://purl.org/dc/terms/",
+    "suggestions": {
+      "@id": "http://www.w3.org/2001/XMLSchema#string",
+      "@container": "@list"
+    }
+  },
+  "@language": "en",
+  "dc:title": "Global top search term",
```

Comment:
I suggest to be more specific in the title and clearer. 
That title raise me the following question:
* "Global" to what? 
* What is the difference between "Global" and "top"? Is it not the same?

Reply:
Christian suggests this title "List of suggestions for the autocomplete function in the top-right search box" if you are agree we will send it for translation. cc @coprisanu

Reply:
@ipaksc do you know the current process on how that file is updated?

That propose name is long. Can we strip out "for the autocomplete function"? Include a qualifier this is for a sites level components. Use the current component name with is "search" and not "top-right search box".
Something like: "List of suggestions for the sites template search component"

\cc @coprisanu @GormFrank